Story/StoryMenu.cs

```diff
@@ -0,0 +1,337 @@
+            for (int i = 0; i < players.Length; i++)
+            {
+                var player = players[i];
+                var btn = new EventfulSelectOneButton(this, mainPage, player.name, "playerButtons", new Vector2(194, 515) - i * new Vector2(0, 38), new(110, 30), playerButtons, i);
+                mainPage.subObjects.Add(btn);
+                playerButtons[i] = btn;
+                btn.OnClick += (_) =>
+                {
+                    string url = $"https://steamcommunity.com/profiles/{player.id}";
+                    SteamFriends.ActivateGameOverlayToWebPage(url);
+                };
+
+            }
+        }
+
+        public override void Update()
+        {
+            base.Update();
+            if (this.rainEffect != null)
+            {
+                this.rainEffect.rainFade = Mathf.Min(0.3f, this.rainEffect.rainFade + 0.006f);
+            }
+
+            ssm.lastScroll = ssm.scroll;
+            ssm.scroll = ssm.NextScroll;
+            if (Mathf.Abs(ssm.lastScroll) > 0.5f && Mathf.Abs(ssm.scroll) <= 0.5f)
+            {
+                this.UpdateCharacterUI();
+            }
+
+            this.clientWaitingButton.buttonBehav.greyedOut = !OnlineManager.lobby.didStartGame;
+
+            if (ssm.scroll == 0f && ssm.lastScroll == 0f)
+            {
+                if (ssm.quedSideInput != 0)
+                {
+                    var sign = (int)Mathf.Sign(ssm.quedSideInput);
+                    ssm.slugcatPageIndex += sign;
+                    ssm.slugcatPageIndex = (ssm.slugcatPageIndex + ssm.slugcatPages.Count) % ssm.slugcatPages.Count;
+                    ssm.scroll = -sign;
+                    ssm.lastScroll = -sign;
+                    ssm.quedSideInput -= sign;
+                    return;
+                }
+            }
+
+
+
+
+        }
+
+
+        private void StartGame()
+        {
+            RainMeadow.DebugMe();
+            manager.arenaSitting = null;
+            manager.rainWorld.progression.ClearOutSaveStateFromMemory();
+            manager.menuSetup.startGameCondition = ProcessManager.MenuSetup.StoryGameInitCondition.New;
+            manager.RequestMainProcessSwitch(ProcessManager.ProcessID.Game);
+        }
+
+        public override void ShutDownProcess()
+        {
+            RainMeadow.DebugMe();
+            if (manager.upcomingProcess != ProcessManager.ProcessID.Game)
+            {
+                MatchmakingManager.instance.LeaveLobby();
+            }
+            base.ShutDownProcess();
+        }
+
+        private void SetupMenuItems()
+        {
+
+            // Host button
+            this.hostStartButton = new EventfulHoldButton(this, this.pages[0], base.Translate("ENTER"), new Vector2(683f, 85f), 40f);
+            this.hostStartButton.OnClick += (_) => { StartGame(); };
+            hostStartButton.buttonBehav.greyedOut = false;
+
+            // TODO: clientWaitingButton needs to not require x/y shift to function. Look into .Remove() on subObjects.
```

Comment:
you might be looking for `MenuObject.RemoveSubObject` (removes child from update tree) and `MenuObject.RemoveSprites` (cleanup graphics), both are needed